src/main/java/com/exasol/adapter/sql/SqlFunctionAggregateGroupConcat.java

```diff
@@ -129,9 +129,16 @@ private Builder(final SqlNode argument) {
          * @return new {@link SqlFunctionAggregateGroupConcat} instance
          */
         public SqlFunctionAggregateGroupConcat build() {
+            validateArgument(this.argument);
             return new SqlFunctionAggregateGroupConcat(this);
         }
 
+        private void validateArgument(final SqlNode argument) {
+            if (argument == null) {
+                throw new IllegalArgumentException("SqlFunctionAggregateGroupConcat has a null argument.");
```

Comment:
is it possible to also indicate in the logs the name of the `SqlFunctionAggregateGroupConcat` ?

Reply:
Do you mean a class name as a variable? I think we will see in the logs from where the exception comes anyway

Reply:
no, I mean the name of the SQL function. For example: "The argument for the `MIN` function can not be NULL"

Reply:
It's a Group Concat function... :D So it's always Group Concat and nothing else